src/v2/guide/components-registration.md

```diff
@@ -94,9 +94,9 @@ new Vue({
 })
 ```
 
-For each property in the `components` object, the key will be the name of the custom element, while the value will contain the options object for the component.
+Para cada propiedad del objeto `components`, la llave será el nombre del elemento personalizado, mientras que el valor contendrá el objeto de opciones del componente.
```

Comment:
I'm pretty sure we are using `clave` instead of `llave` as a translation to `key`.
```suggestion
Para cada propiedad del objeto `components`, la clave será el nombre del elemento personalizado, mientras que el valor contendrá el objeto de opciones del componente.
```